libraries/standard/mqtt/src/mqtt_lightweight.c

```diff
@@ -2027,7 +2027,7 @@ MQTTStatus_t MQTT_DeserializeAck( const MQTTPacketInfo_t * const pIncomingPacket
 
             /* Any other packet type is invalid. */
             default:
-                LogError( ( "IotMqtt_DeserializeResponse() called with unknown packet type:(%lu).", pIncomingPacket->type ) );
+                LogError( ( "IotMqtt_DeserializeResponse() called with unknown packet type:(%u).", pIncomingPacket->type ) );
```

Comment:
I'd prefer this to be changed to hex while we're at it.
```suggestion
                LogError( ( "IotMqtt_DeserializeResponse() called with unknown packet type:(%02x).", pIncomingPacket->type ) );
```

Reply:
Agree. Fixed. Thanks